app/Console/Commands/Migration/RolesMigrationCommand.php

```diff
@@ -28,7 +28,7 @@ class RolesMigrationCommand extends Command
     public function handle()
     {
         if ($this->option('log')) {
-            echo('* * * Started * * * ' . $this->description . chr(10));
+            echo ('* * * Started * * * ' . $this->description . chr(10));
```

Comment:
The linter hasn't completed as of this review, bug I suspect it'll complain about this change. I don't see why this space is needed.

Reply:
space removed